Assets/Scripts/Utils/ColorPallete.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ColorPallete
+{
+    // Implementation based of https://iquilezles.org/articles/palettes/
+    // Health is expected to be normalized.
+    public static Color HealthPalette(float health)
```

Comment:
No needuto specify that it's a palette here

Reply:
Fixed 👍